packages/playground/src/components/node_details.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <v-dialog
-    v-model="dialog"
+    v-model="$props.openDialog"
     @update:modelValue="(val:boolean) => closeDialog(val)"
```

Comment:
I don't think this is a good approach as v-model uses 2-way binding and props are immutable so there might be some issue when we try to mutate it.

Reply:
it is used in many other components safely 
<img width="489" alt="Screenshot 2025-05-04 at 15 30 38" src="https://github.com/user-attachments/assets/3e850164-8ee3-4507-aff9-48f8e643ebcf" />